R_scripts/Generate_warnings.R

```diff
@@ -34,10 +34,11 @@ warning_messages <- c("unhealthy_but_wrong_status" = "There are living tress tha
 
 # check if files exist and generate a plot with the warning ####
 
-if(exists("will_auto_fix_error_file")) all_will_be_fixed <- warning_messages[unique(will_auto_fix_error_file$error_name)]
+all_will_be_fixed <- NULL
+if(file.exists(will_auto_fix_error_filepath)) all_will_be_fixed <- warning_messages[unique(will_auto_fix_error_file$error_name)]
 
-
-if(exists("warning_file")) all_warns <- warning_messages[unique(warning_file$warning_name)]
+all_warns <- NULL
```

Comment:
I added the NULL setting for `all_will_be_fixed` and `all warns` if these variables don't exist, the subsequent
```
png(filename, width = 6, height = 1 + (0.2*length(c(all_warns, all_will_be_fixed))), units = "in", res = 300)
```
line wouldn't work